src/Behat/Gherkin/Lexer.php

```diff
@@ -435,13 +436,24 @@ protected function scanPyStringOp()
             return null;
         }
 
-        if (false === ($pos = mb_strpos($this->line, '"""', 0, 'utf8'))) {
+        if ((false === ($indent = mb_strpos($this->line, $delimiter = '"""', 0, 'utf8')))
+        && (false === ($indent = mb_strpos($this->line, $delimiter ='```', 0, 'utf8')))) {
```

Comment:
what if you have a ` ``` ` followed by `"""` later in the line ? I fear this would parse it wrong

Reply:
I'm not sure what should happen in that case actually

Things like this are ok:
```
"""xml
<content/>
"""
```

I don't know if this should be:

```
""" this is some text ```
foo
"""
```

I guess I can see what the gherkin parser does

Reply:
I guess there's an assumption that the only things before the `"""` should be whitespace, that isn't captured here? Maybe changing this to a regex is the cleanest approach?

Reply:
maybe such case of `"""` not being the first non-whitespace content of the line should be contributed as additional tests in the cucumber tests.